src/components/navbar/navbar.css

```diff
@@ -143,7 +143,7 @@
   }
 
   #half-bg-side {
```

Comment:
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/3c96a683-0e08-4ae4-bc37-04804c518f8e">
<img width="1800" alt="image" src="https://github.com/user-attachments/assets/2fb65e8e-bc6d-4993-9025-ebf2d054ab22">


For a wider view, like on an iPad or something, can you keep the half-panel, and only show the full-panel for Mobile where it makes sense? Maybe use the sm breakpoint?

Or you can have the panel cover the entire screen as you have, but fix everything to the top right of the screen so it always looks like the contents of the nav bar are right-aligned.